packages/gitmoji-changelog-core/src/metaInfo.js

```diff
@@ -4,30 +4,44 @@ const gitRemoteOriginUrl = require('git-remote-origin-url')
 const normalizePackageData = require('normalize-package-data')
 const { isEmpty } = require('lodash')
 
-async function getRepoInfo() {
-  // get the closest package.json file
-  let packageJson = {}
+// get the closest package.json file
+async function getPackageInfo() {
   try {
     const packageInfo = await readPkgUp()
-    packageJson = packageInfo.pkg || {}
+    if (packageInfo.pkg) {
```

Comment:
We could handle negative case first to improve readability 

`if (!packageInfo.pkg) return null`